01.dom_selector/src/selectorTs.ts

```diff
@@ -1,6 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
you can do 
/* eslint-disable @typescript-eslint/no-unused-vars */ 
to disable it on the file level.

Reply:
I know but i choose just for the first line in case the error exist somewhere else in the code (ok not in this example, but in the future)
I will change this if you think its essential

Reply:
iz fine.